model/src/form/form-metadata/index.ts

```diff
@@ -2,8 +2,24 @@ import Joi from 'joi'
 
 import { type FormMetadata } from '~/src/form/form-metadata/types.js'
 
+export const organisations = [
```

Comment:
If you found using array map caused type issues, it's because TypeScript has trust issues

Should help telling it that the array is constant and the item types won't change

```suggestion
export const organisations = /** @type {const} */ ([
```

Reply:
Looks like this wasn't necessary, didn't have squiggles for me. ESLint is happy too